main.js

```diff
@@ -222,7 +222,7 @@ function stopPing(adapter) {
 }
 
 function ping(adapter) {
-    adapter._client.ping && adapter._client.ping(adapter.config.pingInterval - 1000 < 0 ? 1000 : adapter.config.pingInterval - 1000)
+    adapter._client.ping && adapter.config.pingserver && adapter._client.ping(adapter.config.pingInterval - 1000 < 0 ? 1000 : adapter.config.pingInterval - 1000)
```

Comment:
As the pingserver is new option, I would add here `adapter.config.pingserver !== false`